.github/workflows/pull-request.yaml

```diff
@@ -49,13 +49,17 @@ jobs:
             run: ubuntu-22.04
             targets: ci-go-ci-build-linux-static
             go_tags: GO_TAGS="-tags=opa_no_oci"
+            variant_name: opa_no_ci
             arch: arm64
           - os: windows
             run: ubuntu-22.04
             targets: ci-go-ci-build-windows
+            arch: amd64
+          # Note(philipc): The underlying make targets also build arm64_static.
           - os: darwin
             run: macos-latest
             targets: ci-build-darwin ci-build-darwin-arm64-static
+            arch: amd64
```

Comment:
Since this also contains `arm64`, it would have been nice if we could use `all` or something. But I suppose that would mess up the `GOARCH` env var ..

Perhaps we could do a third `Upload Binaries` step to handle the case where `arch` is empty 🤔.

This is very much a nit, so feel free to disregard 😄.

Reply:
In the interest of getting this merged, it may be worth adding a comment for that target about the situation, leaving the `arch: amd64` field alone for now. :thinking: